renderdoc/driver/vulkan/vk_memory.cpp

```diff
@@ -319,9 +319,16 @@ MemoryAllocation WrappedVulkan::AllocateMemoryForResource(bool buffer, VkMemoryR
         break;
     }
 
+    // if ray tracing acceleration structures are in use, then allocate memory with buffer device
+    // address support enabled
+    VkMemoryAllocateFlagsInfo flagsInfo{
```

Comment:
Please use normal initialisation without the omitted =

```suggestion
    VkMemoryAllocateFlagsInfo flagsInfo = {
```